src/Microsoft.AspNet.Mvc.Core/ApplicationModels/DefaultActionModelBuilder.cs

```diff
@@ -266,6 +268,18 @@ private bool IsIDisposableMethod(MethodInfo methodInfo, TypeInfo typeInfo)
             AddRange(actionModel.ActionConstraints, attributes.OfType<IActionConstraintMetadata>());
             AddRange(actionModel.Filters, attributes.OfType<IFilter>());
 
+            var enableCors = attributes.OfType<IEnableCorsMetadata>().SingleOrDefault();
```

Comment:
https://github.com/aspnet/Security/issues/193

I will update the name to `IEnableCorsAttribute`